doxia-modules/doxia-module-xdoc/src/main/java/org/apache/maven/doxia/module/xdoc/XdocParser.java

```diff
@@ -241,6 +246,23 @@ protected void handleEndTag(XmlPullParser parser, Sink sink)
         isEmptyElement = false;
     }
 
+    /** {@inheritDoc} */
+    protected void handleText(XmlPullParser parser, Sink sink) throws XmlPullParserException {
```

Comment:
after having a 2nd thought, this rather seems to be an issue with the XhtmlSink which does not properly trim text following verbatim (with source decoration), as https://maven.apache.org/doxia/doxia/doxia-sink-api/apidocs/org/apache/maven/doxia/sink/Sink.html#verbatim-- is defined as block element (so I think assuming that its content is trimmed seems to be valid). WDYT @michael-o ?